src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { forwardRef } from 'react';
+import { bem } from '../../utils';
+import styles from './RadioButtonGroup.scss';
+
+export interface Props {
+    /** The name of the group this radio button belongs to */
+    name: string;
+    /** common onChange handler that will be passed to all children */
+    onChange?: () => void;
+    /** The radio buttons */
+    children: React.ReactElement[];
```

Comment:
Should `children` only accept an array of React elements? Let's use one of the type from `types.d.ts`.

Reply:
From the original implementation it doesn't contain isRequired 
![image](https://user-images.githubusercontent.com/104063210/202469222-e2fdeaf9-690a-4c90-960d-7b9a06d307a7.png)